dotnet/src/messages/BasePublicEvent.cs

```diff
@@ -0,0 +1,33 @@
+
+namespace ProcessEngine.ConsumerAPI.Contracts.Messages
+{
+    /// <summary>
+    /// The base class for definining event messages.
+    /// </summary>
+    public class BasePublicEventMessage
```

Comment:
```suggestion
    public class BaseEventMessage
```

Ther's no need for that `Public` anymore.